Remove Rubocop's comments from Rails code base

bogdanvlviv opened the pull request:
PR#32381 added Rubocop's comments to some tests files in order to
exclude `Performance/RedundantMerge`.

Turn off `Performance` cops for tests files via `Exclude`
in `.rubocop.yml`.

<hr/>
Fix Rubocop offense

```
Offenses:

activesupport/lib/active_support/subscriber.rb:91:17:
C: Layout/SpaceAroundOperators: Operator = should be surrounded by a single space.
      event     = event_stack.pop
```

bogdanvlviv committed:
Remove Rubocop's comments from Rails code base

PR#32381 added Rubocop's comments to some tests files in order to
exclude `Performance/RedundantMerge`.

Turn off `Performance` cops for tests files via `Exclude`
in `.rubocop.yml`.

Context https://github.com/rails/rails/pull/32381#discussion_r205212331

kaspth commented on .rubocop.yml:
```diff
@@ -188,3 +188,6 @@ Performance/FlatMap:
   Enabled: true
+  Exclude:
+    - 'actionpack/test/dispatch/header_test.rb'
+    - 'activerecord/test/cases/relation_test.rb'
```
What would it look like if we excluded every test from these performance cops?

bogdanvlviv committed:
Fix Rubocop offense

```
Offenses:

activesupport/lib/active_support/subscriber.rb:91:17:
C: Layout/SpaceAroundOperators: Operator = should be surrounded by a single space.
      event     = event_stack.pop
```

bogdanvlviv commented:
> What would it look like if we excluded every test from these performance cops?

@kaspth Seems reasonable to exclude `Performance` cops for tests. I just updated this PR to do that.

I also added one more commit in order to fix rubocop offense:
```
activesupport/lib/active_support/subscriber.rb:91:17:
C: Layout/SpaceAroundOperators: Operator = should be surrounded by a single space.
      event     = event_stack.pop
```
